.idea/misc.xml

```diff
@@ -5,7 +5,7 @@
       <item index="0" class="java.lang.String" itemvalue="org.bukkit.event.EventHandler" />
     </list>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_3" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
You need to check your settings. Your JDK version is set to 1.3 somewhere and then your ignoring it somewhere else. Just set it to 1.8 for everything